adrs/php/0002_module_structure.md

```diff
@@ -0,0 +1,29 @@
+# Module structure
+
+## Motivation
+
+Standardize the design of the different modules. Considering a module as an internal package that contains
+a group of related classes that belong together driven by its scope.
+
+## Decision
+
+### Follow Package design
+
+- Coupling is the degree of interdependence between software modules.
+- Cohesion refers to how the elements of a module belong together.
+
+We want to design components that are self-contained: independent, and with a single, well-defined purpose.
+We stand for low coupling and high cohesion.
+
+### Module standard design
+
+The modules have a similar structure between each other. At the root of a module you can find:
+
+- Facade: A Facade is the class that defines the entry point of the module.
+  - There must be one Facade per module.
```

Comment:
"exactly one Facade" or "at least one Facede"?

Reply:
"exactly one Facade". I will change that :)

Reply:
I am thinking about "There can be exactly one Facade per module".

I would say this idea is still WIP, because what we are using in some modules (still) is the Factory to communicate the modules and not the Facade. This means that we have still modules without Facade at all. So it can be confusing seeing "must be exactly one Facade" and then find none :P 

 For this reason, I will use "can" instead of "must" in this sentence. Ok with this?